### DISCUSSION TITLE
Make multi condition

### DISCUSSION THREAD

**Discussion by benjibenoit** [directus/directus#14515] [Archive]
Hi,

I need to know how I can do two conditions at the same time on one table?

Let's imagine the table below. How can I make sure that in the "like" table, a user can only like a comment once. The row (comment id and user id) should be unique.

Is it possible to do this?

Thanks 

![image](https://user-images.githubusercontent.com/32774804/179799294-8bc60c43-c8c3-4fad-9acb-71524929757e.png)

**Comment by rijkvanzanten** [comment]
Directus doesn't currently support composite unique keys. https://github.com/directus/directus/discussions/14158 is the open feature request for that 🙂
